### ISSUE TITLE
Can you create a testing branch ?

### ISSUE THREAD

**Issue by yalh76** [YunoHost-Apps/yunorunner_ynh#1]
I have a small PR to push ^^

**Comment by maniackcrudelis** [comment]
You can make a pull request on master, anyway, no one is using a testing branch for this app. So no one is going to test it.
So, make your pull request directly to master, it will be fine.

**Comment by yalh76** [comment]
Done.
I think I will do another PR to add arguments for the installation like : 
1. Public or not
2. Monitor the app list or not
3. Providing the path for analyseCI.sh
These options can let packagers have their own YunoRunner CI

A question not far.. Is it a good idea to package https://github.com/YunoHost/CI_package_check ?
To be able to install a complete Yunohost CI 
And if "yes": what name to use: ci_package_check_ynh or ci_package_check_ynh_core ?

**Comment by maniackcrudelis** [comment]
Huh yep if you want you can add those arguments to the manifest, but the builders in CI_package_check will need to be updated as well.

Globally, about packaging the CI, there's some points to consider:

- YunoRunner is still a work in progress.
- Neither YunoRunner or CI_package_check are safe to use and without bugs.
- We're not working on these 2 apps considering final user usage. Our work is aimed toward our official CI.

If you run YunoRunner or CI_package_check directly, you'll probably trig every github commits for both official and community apps.
For packagers usage, I would recommend using dev_CI scripts in CI_package_check with jenkins.
Or even directly using jenkins and add separately your jobs.
A topic on the forum explain how to deploy jenkins to test your own apps. https://forum.yunohost.org/t/integration-continue-sur-les-packages-dapplications-yunohost/2219

**Comment by yalh76** [comment]
For the arguments, I propose to use the arguments already available in CI_package_check

About packaging the two apps, I'm just thinking that the more packagers will use them on their own, the more improvements and help can be done on this two apps... Even if they are not as important as other apps.

For now, I've installed Yunorunner / CI_package_check / Jenkins on the same VM and I'm able to launch jobs from Yunnorunner and from Jenkins (using https://forum.yunohost.org/t/integration-continue-sur-les-packages-dapplications-yunohost/2219). I have implemented in Jenkins some jobs to build from master and testing branches for the packages I work for.

About Arm, we discuss in another conversation, I've just installed an ARM VM on a Debian 64 stretch/Proxmox: https://forum.proxmox.com/threads/arm64-vm-emulation-on-proxmox-5-3-amd64.49699/. Next step: installing yunohost / CI_package_check to also be able to configure jobs form ARM ^^
